docs/cli/vast-export-json.md

```diff
@@ -1,2 +1,7 @@
 The JSON export format renders events in newline-delimited JSON (aka.
 [JSONL](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON)).
+
+By default, durations render as human-readable strings with up to two decimal
+places and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
```

Comment:
```suggestion
By default, durations render as human-readable strings rounded up to two decimal
digits and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
```

Reply:
"rounded up" can be taken quite literal.

Reply:
What is the issue with this?

Aside from it, we should say simply "rounded to..." without "up".

Reply:
Isn't "decimal places" correct over "decimal digits"? I've never heard the latter.

Reply:
I think "place" is indeed more common. Personally, I find "digit" more visual and easier to grasp, but then this doesn't matter if "decimal place" is the common expectation.

Consider my feedback on this as discarded.